torch/_inductor/codegen/rocm/rocm_kernel.py

```diff
@@ -211,6 +214,21 @@ def size(
         val = sympy_product(sizes)
         return cexpr(self.rename_indexing(val))
 
+    def contiguous_stride(self, node: IRNode, default_value: int = 0) -> str:
```

Comment:
Nit: maybe `get_contiguous_stride`?

Reply:
but: size() and stride() in the CUTLASS backend. I think 'get' is sort of implied here and there